### DISCUSSION TITLE
First App ... does not show - cannot be created

### DISCUSSION THREAD

**Discussion by tetter123** [Budibase/budibase#4152] [Help]
I just wanted to start looking at budibase ... but event the first action did not work - create an app :((

I gives me an error "Error: Document update conflict" ... and nothing more ... basically this is unusable ... and the same happend 2 weeks ago.

I use the online hosted sandbox.

**Comment by mjashanks** [comment]
Hey @tetter123 thanks for letting us know. We'd like to investigate this more. Could you let use know your budibase domain name... i.e. `https://your-budibase-name.budibase.app` - feel free to send this to support@budibase.com if you don't want to share it publically.

It would also be good to know
1. What browser and OS are you using?
2. Are you using it inside a company firewall?

Thank you!
